eclair-core/src/main/resources/reference.conf

```
@@ -114,6 +114,16 @@ eclair {
       ratio-low = 0.5 // will allow remote fee rates as low as half our local feerate
       ratio-high = 10.0 // will allow remote fee rates as high as 10 times our local feerate
     }
+    override-feerate-tolerance = [ // optional per-node feerate tolerance
+      //      {
+      //        nodeid = "02aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+      //        feerate-tolerance {
+      //          ratio-low = 0.1
+      //          ratio-high = 20.0
+      //        }
+      //      }
```

Nit, for consistency with `override-features`:

```suggestion
      #      {
      #        nodeid = "02aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
      #        feerate-tolerance {
      #          ratio-low = 0.1
      #          ratio-high = 20.0
      #        }
      #      }
```

---

Done in https://github.com/ACINQ/eclair/pull/1575/commits/463f39a919ef93f525095441f6449ba1090612a7